pkg/controller/velero/velero.go

```diff
@@ -33,8 +33,7 @@ const (
 	awsCredsSecretIDKey     = "aws_access_key_id"     // #nosec G101
 	awsCredsSecretAccessKey = "aws_secret_access_key" // #nosec G101
 
-	veleroImageRegistry   = "docker.io/velero"
-	veleroImageRegistryCN = "registry.docker-cn.com/velero"
+	veleroImageRegistry = "quay.io/app-sre"
```

Comment:
If we're gonna do this, we need to pull the image by digest rather than by tag on [L360](https://github.com/openshift/managed-velero-operator/pull/126/files#diff-c2dd6e43d0ffc4b8494e0ee0e68305942c89786301a2891ff562084e8d972446R360] and [L367](https://github.com/openshift/managed-velero-operator/pull/126/files#diff-c2dd6e43d0ffc4b8494e0ee0e68305942c89786301a2891ff562084e8d972446R367)

Which means whoever is building/pushing images to quay.io/velero/velero needs to make sure they don't push the same image twice.

I can't find that repo -- does it not exist yet?

Reply:
Well crap. The plugin images also need to be pulled by digest, which *would* be [L359](https://github.com/openshift/managed-velero-operator/pull/126/files#diff-c2dd6e43d0ffc4b8494e0ee0e68305942c89786301a2891ff562084e8d972446R359) and [L366](https://github.com/openshift/managed-velero-operator/pull/126/files#diff-c2dd6e43d0ffc4b8494e0ee0e68305942c89786301a2891ff562084e8d972446R366) -- but `WithPlugins` ends up parsing the image URI via [this `getName` method](https://github.com/vmware-tanzu/velero/blob/v1.6.0/pkg/builder/container_builder.go#L46-L71), which isn't currently digest-aware.

Reply:
Its actually quay.io/app-sre/velero, and they're private repos. They're configured here: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/19345

Reply:
FWIW, I proposed https://github.com/vmware-tanzu/velero/pull/3803 to add support for this.

Reply:
> Its actually quay.io/app-sre/velero, and they're private images. They're configured here: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/19345

Cool. If they're cloned & controlled by app-sre, we should be okay wrt digests not changing, as they're aware of this issue for other similar clones.